pkg/storage/tsdb/bucketindex/loader.go

```diff
@@ -24,6 +25,7 @@ import (
 )
 
 type LoaderConfig struct {
+	ExtraMetricsPrefix    string
```

Comment:
this avoids duplicate metrics registration when we have multiple bucket index loaders initialized